tests/src/de/tomgrill/gdxtesting/TutorialTranslationTests.kt

```diff
@@ -0,0 +1,47 @@
+package de.tomgrill.gdxtesting
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.utils.Array
+import com.badlogic.gdx.utils.Json
+import com.unciv.models.Tutorial
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(GdxTestRunner::class)
+class TutorialTranslationTests {
+
+    private val filesPath = "jsons/Tutorials/Tutorials_%s.json"
+    private var tutorialCount = Tutorial.values().size
+    private var tutorialKeyNames = Tutorial.values().map { it.value }
+    private val json = Json().apply { ignoreUnknownFields = true }
+
+    private val languages = listOf(
+            "English", "Czech", "French", "Italian",
+            "Korean", "Polish", "Ukrainian", "Russian",
+            "Simplified_Chinese", "Traditional_Chinese"
+    )
+
+    @Test
+    fun testValidNumberOfTranslations() {
+        languages.forEach { language ->
```

Comment:
I remember doing something like this and encountering that having more than one assertTrue in a function would mean the test woudn't work properly, does this run OK for you?

Reply:
yes it works fine
the main point of this is if it will fails on `n` iteration it break all this test without iterating all list